apps/prairielearn/src/lib/externalGrader.ts

```diff
@@ -160,11 +159,11 @@ async function updateJobReceivedTime(grading_job_id: string, receivedTime: strin
  */
 export async function processGradingResult(content: any): Promise<void> {
   try {
-    if (!_.isObject(content.grading)) {
+    if (content.grading == null || typeof content.grading !== 'object') {
       throw new error.AugmentedError('invalid grading', { data: { content } });
     }
 
-    if ('feedback' in content.grading && !_.isObject(content.grading.feedback)) {
+    if (content.grading.feedback != null && typeof content.grading.feedback !== 'object') {
```

Comment:
I wonder if perhaps we actually want the behavior provided by `_.isPlainObject` here if we want to maintain the property that `feedback` is an object.

Reply:
Done in fcfccbfc789bfff4857a47903ae6e309d697c0dd.